template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/databricks.yml.tmpl

```diff
@@ -12,10 +12,10 @@ variables:
     {{- else -}}default: {{template `model_name` .}}{{end}}
 
 include:
-  # Resources folder contains ML artifact resources for the ml project that defines model and experiment
-  # And workflows resources for the ml project including model training -> validation -> deployment,
+  # assets folder contains ML artifact assets for the ml project that defines model and experiment
```

Comment:
Can we be sure to capitalize where `Resources` was capitalized? For a search and replace, we can first do a case-sensitive query and then an insensitive one

Reply:
yep, corrected the capitalized `Assets`